packages/p2p/src/components/modal-manager/modals/buy-sell-modal/buy-sell-modal-error/buy-sell-modal-error.tsx

```diff
@@ -11,19 +12,16 @@ const BuySellModalError = ({ error_message, show_low_balance_message }: TBuySell
     if (error_message || show_low_balance_message) {
         return (
             <div className='buy-sell-modal-error'>
-                <HintBox
-                    className='buy-sell-modal-error__danger'
-                    icon='IcAlertDanger'
+                <InlineMessage
                     message={
-                        <Text as='p' size='xxxs' color='prominent' line_height='s'>
-                            {show_low_balance_message ? (
-                                <Localize i18n_default_text="Your Deriv P2P balance isn't enough. Please increase your balance before trying again." />
-                            ) : (
-                                error_message
-                            )}
-                        </Text>
+                        show_low_balance_message
+                            ? localize(
+                                  "Your Deriv P2P balance isn't enough. Please increase your balance before trying again."
+                              )
+                            : error_message
```

Comment:
why are we changing this?

Reply:
because inside of the InlineMessage component, the message being sent is wrapped inside of text, would it be better to pass it as Localize instead?

Reply:
please add this to QA checklist if this is within the scope of the card

Reply:
sure

Reply:
I have already referenced to check all the different occurences of the hint boxes 👍